src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -108,14 +107,18 @@ void AppleseedRenderer::DeleteThis()
 
 void* AppleseedRenderer::GetInterface(ULONG id)
 {
+#if MAX_RELEASE == MAX_RELEASE_R18  // No interactive render for max 2015.
```

Comment:
What about Max 2017 and later?

Reply:
2017 and later use special methods instead of GetInterface.